consensus/merge/src/test/java/org/hyperledger/besu/consensus/merge/headervalidationrules/NoDifficultyValidationTest.java

```diff
@@ -56,6 +56,7 @@ public void difficultyMustBeZero() {
 
     final BlockHeader invalidHeader = mock(BlockHeader.class);
     when(invalidHeader.getDifficulty()).thenReturn(Difficulty.of(10));
+    when(invalidHeader.getNumber()).thenReturn(1337L);
```

Comment:
Tell me it's Gary without telling me it's Gary :)

Reply:
lol